public/views/children.html

```diff
@@ -46,7 +46,10 @@
                     <div class="col-sm-2 pre-active" style="text-align: center" >
                         <a style="text-align: center"  href="/views/children.html">Apadrina</a>
                     </div>
-                    <div class="col-sm-2 pre-active">
+                    <div class="col-sm-1" style="text-align: center" >
```

Comment:
you should make this col-md-2 and add the pre-active class so that it's orange and not blue